src/PrestaShopBundle/ApiPlatform/Provider/QueryProvider.php

```diff
@@ -47,9 +47,15 @@ public function __construct(
     }
 
     /**
-     * @throws \ReflectionException
-     * @throws \Exception
+     * @param Operation $operation
+     * @param array $uriVariables
+     * @param array $context
+     *
+     * @return \ApiPlatform\State\Pagination\PartialPaginatorInterface|array|\ArrayObject|bool|float|int|mixed|string|null
```

Comment:
Is it useful to mark it can return quite everything? 😛

Reply:
return mixed will be sufficient :D